### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Office Scripts documentation
 
-Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you view this content on [docs.microsoft.com](https://docs.microsoft.com/office/dev/scripts).
+Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you review the [Office Scripts documentation](https://docs.microsoft.com/office/dev/scripts).
```
Comment (AlexJerabek) [inline README.md:3]: This changes the meaning of the line. This GitHub readme is intended to tell people not to view the docs in GitHub itself, but rather go to the rendered DMC site.

Comment (alexbuckgit) [reply README.md:3]: That makes sense. "docs.microsoft.com" is supposed to be avoided as sentence text or as a link description - can you think of a different phrase we could use here that wouldn't change the meaning?

#docutune-wait

**Comment by alexbuckgit** [comment]
#docutune-wait

File: README.md (hunk 1 above)
Comment (AlexJerabek) [reply README.md:3]: Thanks for understanding and explaining the changes. How about something like this?
```suggestion
Welcome to the Office Scripts documentation repository. In this repo, you can find the documentation source files for Office Scripts tutorials and how-to guides. For the best experience, we recommend you view this content on [Microsoft Docs](https://docs.microsoft.com/office/dev/scripts).
```
(I'll be sure to change the text once the Learn rebrand happens.)

**Comment by brittmsantos** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@AlexJerabek

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Pull request merged**